react/FilePicker/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import PropTypes from 'prop-types'
+
+import { FixedDialog } from '../CozyDialogs'
+import FilePickerHeader from './FilePickerHeader'
+import FilePickerFooter from './FilePickerFooter'
+import FilePickerBody from './FilePickerBody'
+
+const getIsAccept = accept => {
+  // If accept is falsy, set default to file
+  const acceptSelection = accept
+    ? accept.replaceAll(' ', '').split(',')
+    : ['file']
+
+  return {
+    file: acceptSelection.includes('file'),
+    folder: acceptSelection.includes('folder')
+  }
+}
+
+const useStyles = makeStyles(theme => ({
+  paper: {
+    height: '100%',
+    '& >div:nth-child(4)': {
```

Comment:
This rule is a bit cryptic, without starting the app I have no idea of what is the 4th paper's child nor why a padding 0 is set to it.

Is is possible to have a more semantic rule? If not can you add a comment explaining what is happening here?

Reply:
This code does not exist anymore, this PR has solved this problem. https://github.com/cozy/cozy-ui/pull/2009